content/en/docs/contribute/style/style-guide.md

```diff
@@ -110,8 +110,16 @@ document, use the backtick (`).
   <tr><th>Do</th><th>Don't</th></tr>
   <tr><td>The <code>kubectl run</code> command creates a Deployment.</td><td>The "kubectl run" command creates a Deployment.</td></tr>
   <tr><td>For declarative management, use <code>kubectl apply</code>.</td><td>For declarative management, use "kubectl apply".</td></tr>
+  <tr><td>Enclose code samples with triple backticks. <code>(```)</code></td><td>Enclose code samples with any other syntax.</td></tr>
 </table>
 
+{{< note >}} 
+The website supports syntax highlighting for code samples, but specifying a language isn't required, is optional. For example, you can enable highlighting for Python samples by enclosing the sample like so: 
```

Comment:
@DanyC97 One final nit. Let's use this wording instead:
> The website supports syntax highlighting for code samples, but specifying a language is optional.
Otherwise LGTM 👍

Reply:
done @zacharysarah

Reply:
@DanyC97 Be sure to remember the period at the end. 😉